groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -480,6 +480,17 @@ groups:
       - dlipovetsky@d2iq.com
       - neolit123@gmail.com
 
+  - email-id: k8s-infra-staging-kubespray@kubernetes.io
```

Comment:
> Could you assist the Kubespray team in establishing a notification mailing list for Kubespray-related communications?

in that case you might to scope and name the ML differently.

i'm proposing the name `sig-cluster-lifecycle-kubespray`